downstream/attributes/attributes.adoc

```diff
@@ -277,94 +277,124 @@
 //:MenuSetLogin: {MenuAEAdminSettings}[Log In Settings]
 //:MenuSetUI: {MenuAEAdminSettings}[User Interface Settings]
 
-// Title attributes
+// Title and link attributes
 //
 // titles/troubleshooting-aap
 :TitleTroubleshootingAAP: Troubleshooting Ansible Automation Platform
+:LinkTroubleshootingAAP: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/troubleshooting_ansible_automation_platform
 //
 // titles/aap-plugin-rhdh-install
 :TitlePluginRHDHInstall: Installing Ansible plug-ins for Red Hat Developer Hub
+:LinkPluginRHDHInstall: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/installing_ansible_plug-ins_for_red_hat_developer_hub
 //
 // titles/aap-plugin-rhdh-using
 :TitlePluginRHDHUsing: Using Ansible plug-ins for Red Hat Developer Hub
+:LinkPluginRHDHUsing: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/using_ansible_plug-ins_for_red_hat_developer_hub
```

Comment:
The hyphen is correct: I checked it against the [2.4 doc on the portal](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html/installing_ansible_plug-ins_for_red_hat_developer_hub/index)